thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/health/DetectionTaskStatus.java

```diff
@@ -46,10 +46,14 @@
 
   // the time stamp of last successfully finishing task
   @JsonProperty
-  private final Long lastTaskExecutionTime;
+  private Long lastSuccessTaskExecutionTime;
```

Comment:
We may need to keep the name, otherwise, the existing JSON blobs in the DB won't be able to deserialize back to the object.

Reply:
Reverted.